style.css

```diff
@@ -0,0 +1,118 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+    
+}
+
+body {
+    background-color: #f0f4f8;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    color: #333;
+}
+
+header {
+    background-color: #4185d1;
+    color: white;
+    padding: 20px 0;
+    text-align: center;
+}
+
+
+.main_container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 50px 10%;
+    height: 80vh;
+    
+}
+
+.form_container {
+    background: #ffffff;
+    padding: 1rem;
+    border-radius: 8px;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.1);
+    width: 60%;
+    height: 50vh;
+}
+.resources_container {
+    background: #ffffff;
+    padding: 1rem;
+    border-radius: 8px;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.1);
+    width: 35%;
+    height: 50vh;
+}
+
+h2{
+    margin-bottom: 1vh;
+}
+
+h3 {
+    margin-bottom: 2vh;
+}
+
+
+form {
+    display: flex;
+    flex-direction: column;
+}
+
+label {
+    font-weight:500;
+    margin-bottom: 10px;
+}
+
+input, textarea {
+    margin-bottom: 3%;
+    padding: 2%;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    font-size: 16px;
+}
+
+button {
+    background-color: #ff5722;
+    color: white;
+    padding-top: 0.75%;
+    padding-bottom: 0.75%;
+    border-radius: 4px;
+    font-size: 15px;
+    border: none;
+}
+
+button:hover {
+    background-color: #e64a19;
+}
+
+ul {
+    list-style-type: none; /* elimina les boles de la llista */
```

Comment:
This rule should be moved to `sanitize`, so you will ensure all `ul` elements will be overwritten with this rule.

Remove comment.